### ISSUE TITLE
`PascalCase` and `snake_case`

### ISSUE THREAD

**Issue by seb-jean** [VincentLanglet/Twig-CS-Fixer#204]
### Rule(s) related to or rule(s) missing

- DirectoryNameRule
- FileNameRule

### Expected behavior

Use `PascalCase`.

### Actual behavior

I use `snake_case` for all my Twig files. But exceptionally in a folder, I also use another case, the `PascalCase`. I use the `PascalCase` because I use [Twig Components](https://symfony.com/bundles/ux-twig-component/current/index.html). The files are then of the following form:
- templates/components/Alert.html.twig
- templates/components/Alert/Succes.html.twig
- ...

For example, here is an error I have:

 KO .\templates\components\Text\TextLink.html.twig
 ------- ---------------------------------------------------------------
  ERROR   >>   | The file name must use snake_case; expected text_link.
 ------- ---------------------------------------------------------------
  ERROR   >>   | The directory name must use snake_case; expected text.
 ------- --------------------------------------------------------------

```php
// .twig-cs-fixer.php

<?php

$ruleset = new TwigCsFixer\Ruleset\Ruleset();
$ruleset->addStandard(new TwigCsFixer\Standard\Symfony());

$config = new TwigCsFixer\Config\Config();
$config->setRuleset($ruleset);
$config->setCacheFile(null);
$config->allowNonFixableRules();

return $config;
```

**Comment by VincentLanglet** [comment]
You gave the wrong config file, this is not FriendsOfTwig.

The `case` rules have configuration see for example how it works for SymfonyStandard
https://github.com/VincentLanglet/Twig-CS-Fixer/blob/main/src/Standard/Symfony.php#L24-L25

You can do
```
new FileNameRule(baseDirectory: 'templates', ignoredSubDirectories: ['component', 'bundles'], optionalPrefix: '_'),
new DirectoryNameRule(baseDirectory: 'templates', ignoredSubDirectories: ['component', 'bundles']),
```
for instance.

**Comment by seb-jean** [comment]
I edited with the correct content.

**Comment by VincentLanglet** [comment]
Can you try the https://github.com/VincentLanglet/Twig-CS-Fixer/pull/205 branch @seb-jean ?

**Comment by seb-jean** [comment]
It's good :)

**Comment by VincentLanglet** [comment]
Released in 2.6.0
